modules/rostests/kmtests/ntos_mm/MmReservedMapping.c

```diff
@@ -187,6 +213,7 @@ TestMap(
 
         KmtStartSeh()
             *(volatile ULONG *)BaseAddress = 0x01234567;
+            ok_eq_ulong(*(volatile ULONG *)BaseAddress, 0x01234567);
```

Comment:
That's interesting. What could make it not succeed? A non-writable base address?